docs/dev/table_api.md

```diff
@@ -1656,6 +1656,29 @@ temporalOverlaps(TIMEPOINT, TEMPORAL, TIMEPOINT, TEMPORAL)
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight java %}
+ANY.flatten()
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Converts a Flink composite type (such as Tuple, POJO, etc.) and all of its subtypes into a flat representation where every subtype is a separate field.</p>
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        {% highlight java %}
+COMPOSITE.get(STRING)
```

Comment:
I thought we only agreed that `getField` is too long. Personally I like this approach more because it can be found in the expression DSL and has a Scala doc; and it doesn't add further case distinction logic to the validation layer. Since "$" is valid part of Java identifier we would also restrict the field naming of POJOs. If there is a consensus I can change it to "field$substring".